test/TempProbe_TC.cpp

```diff
@@ -0,0 +1,155 @@
+#include "Arduino.h"
```

Comment:
I know that I originally suggested that you put the "wrapper" class in with the tests, but now I'd like it submitted as a separate PR to `TankControllerLib` in the `devices` directory.